tests/data/pluto_disk_planet/definitions.h

```diff
@@ -28,11 +28,8 @@
 /* [Beg] user-defined constants (do not change this line) */
 
 #define  LIMITER                        VANLEER_LIM
-#define  UNIT_LENGTH                    7.779090384e13
-// #define  UNIT_LENGTH                    (5.2*CONST_au)
-#define  UNIT_DENSITY                   4.24857748e-9
-// #define  UNIT_DENSITY                   (CONST_Msun/(UNIT_LENGTH*UNIT_LENGTH*UNIT_LENGTH))
-#define  UNIT_VELOCITY                  208457.85579
-// #define  UNIT_VELOCITY                  (sqrt(CONST_G*g_inputParam[Mstar]*CONST_Msun/UNIT_LENGTH)/(2.*CONST_PI))
+#define  UNIT_LENGTH                    (5.2*CONST_au)
+#define  UNIT_DENSITY                   (CONST_Msun/(UNIT_LENGTH*UNIT_LENGTH*UNIT_LENGTH))
+#define  UNIT_VELOCITY                  (sqrt(CONST_G*g_inputParam[Mstar]*CONST_Msun/UNIT_LENGTH)/(2.*CONST_PI))
```

Comment:
are we testing that a the result of parsing is unchanged ?

Reply:
Yes. Due to such definitions cannot be resolved before, I left their numerical results which are given in `tests/data/pluto_disk_planet/meta.yaml`

Reply:
excellent, thanks !